manifests/init.pp

```diff
@@ -43,6 +43,7 @@
   Optional[Stdlib::Absolutepath] $ssl_cert = undef,
   Optional[Stdlib::Absolutepath] $ssl_key = undef,
   Optional[String] $ssl_protocol = undef,
+  Optional[Boolean] $manage_service = undef,
```

Comment:
Instead of this, can you use Boolean as datatype? Wrapping Boolean in Optional is an antipattern.

Reply:
No. If I change this, the pick statement in prosody::service does not work anymore !
See this parameter as an overwrite of $daemonize. undef means, we take the value of daemonize. This is to ensure we have the same behaviour as in previous versions.